tests/eosio.rentbw_tests.cpp

```diff
@@ -331,7 +436,7 @@ struct rentbw_tester : eosio_system_tester
       auto after_reserve = get_account_info(N(eosio.reserv));
       auto after_state = get_state();
 
-      if (csv)
+      if (csv && max <= 0)
```

Comment:
You don't want to log anything if `queue_max` is bigger than 0. Is it right?

Reply:
I realized that I want to log all actions. I have reverted that change.